CHANGELOG.md

```diff
@@ -72,6 +84,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - Local file read/write support.
 
+[4.0.1]: https://github.com/resin-io/resin-image-fs/compare/v3.0.0...v4.0.1
```

Comment:
oops, I'll fix this ASAP, I just need to update file-disk in partitioninfo first.
The build fails because the file-disk version in partitioninfo and resin-image-fs is different so `xxx instanceof filedisk.Disk` failse.